webui/react/src/hooks/useModal/HyperparameterSearch/useModalHyperparameterSearch.tsx

```diff
@@ -109,9 +109,10 @@ const useModalHyperparameterSearch = ({
     Object.values(SEARCH_METHODS).find((searcher) => searcher.name === experiment.searcherType) ??
       SEARCH_METHODS.ASHA,
   );
-  const { resourcePools } = useStore();
+  const { resourcePools: loadableResourcePools } = useResourcePools();
+  const resourcePools = Loadable.getOrElse([], loadableResourcePools);
   const canceler = useRef<AbortController>();
-  const fetchResourcePools = useFetchResourcePools(canceler.current);
+  const fetchResourcePools = useFetchResourcePools(canceler.current ?? new AbortController());
```

Comment:
nit: seems like this could be simplified by setting the initial value of the canceler useRef above to `new AbortController()` and removing `canceler.current = new AbortController();` in the useEffect on line 131